src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -100,7 +98,7 @@ export const SchemaDrivenAnnotationEditor = (
   const ANNOTATION_EDITOR_TOOLTIP_ID = 'AnnotationEditorTooltip'
 
   const { entityMetadata: entityJson, annotations } = useGetJson(entityId!, {
-    enabled: !entityId || !formData, // once we have data, don't refetch. it would overwrite the user's entries
+    enabled: !!entityId && !formData, // once we have data, don't refetch. it would overwrite the user's entries
```

Comment:
I think this was a bug--we would attempt to fetch entity JSON even if if entityId was undefined. Should fetch the JSON if we have an entity ID and we don't have form data.